windows/cross-compile/build.sh

```diff
@@ -883,14 +883,14 @@ fi
 # Toxcore
 
 TOXCORE_PREFIX_DIR="$DEP_DIR/libtoxcore"
-TOXCORE_VERSION=0.1.11
+TOXCORE_VERSION=0.2.1
 TOXCORE_HASH=56a4566a5cdd201ef27d3a884c0f9aa17e99a300f08ace6323306d24928e727f
```

Comment:
New hash is `1496164954941b175493fba02bf3115118c0d29feb46cd1ff458a1a11eab1597`

Reply:
damn bash scripts, done